test/IISIntegration.FunctionalTests/NtlmAuthentationTest.cs

```diff
@@ -22,29 +21,23 @@ public NtlmAuthenticationTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        [ConditionalTheory]
-        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
-        [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR)]
-        [InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x64, ApplicationType.Portable)]
-        public Task NtlmAuthentication(RuntimeFlavor runtimeFlavor, RuntimeArchitecture architecture, ApplicationType applicationType)
```

Comment:
Gotcha. Could I use `#ifdef` instead so we get mutually exclusive sets of tests - things that run using CoreCLR and things that run using CLR?